lib/pdksync.rb

```diff
@@ -139,36 +139,46 @@ def self.main(steps: [:clone], args: nil)
       if steps.include?(:push)
         Dir.chdir(main_path) unless Dir.pwd == main_path
         git_instance = Git.open(output_path)
-        push_staged_files(git_instance, git_instance.current_branch, repo_name)
-        print 'push, '
+        # replace 'origin/master' with proper upstream default branch when https://github.com/ruby-git/ruby-git/issues/414 gets fixed
+        if git_instance.diff(git_instance.current_branch, 'origin/master').size != 0 # doesn't have a empty? method # rubocop:disable Style/ZeroLengthPredicate
```

Comment:
As opposed to `origin/master` shouldn't this be `#{@push_file_destination}/#{@create_pr_against}` since the PR destination is part of the config?

Reply:
good catch.